nengo_viz/swi.py

```diff
@@ -402,14 +408,18 @@ def start(cls, port=80, asynch=True, addr='', browser=False):
             # shut down any remaining threads
             if asynch and server.requests is not None:
                 for _, sock in server.requests:
-                    sock.close()
+                    try:
+                        sock.shutdown(socket.SHUT_RDWR)
```

Comment:
Hah, I tried both `shutdown` and `close`, but I totally missed that line in the docs about calling `shutdown` *before* `close`.